NSSLServer/Program.cs

```diff
@@ -1,53 +1,149 @@
-﻿using NLog.Web;
-
+﻿using Microsoft.AspNetCore.Mvc.ApplicationParts;
+using Microsoft.AspNetCore.ResponseCompression;
+using Microsoft.OpenApi.Models;
 using NSSLServer.Features;
+using Serilog;
+
+ThreadPool.SetMaxThreads(500, 500);
+
+Log.Logger = new LoggerConfiguration()
+    .MinimumLevel.Verbose()
+    .WriteTo.File("external/logs/log.txt", rollingInterval: RollingInterval.Day, retainedFileCountLimit: 7)
+    .WriteTo.Console()
+    .Enrich.FromLogContext()
+    .CreateLogger();
```

Comment:
Das hier sollte eher in eine Config File ausgelagert werden, weil ich Verbose Files eher wo anders hinschreiben möchte z.B.
Zumindest hoffe ich, dass das mit Serilog auch noch geht, dass man mehrere Files hat